ServerCore/Pages/Hints/CreateBulkMulti.cshtml

```diff
@@ -0,0 +1,75 @@
+﻿@page "/{eventId}/{eventRole}/Hints/CreateBulkMulti"
```

Comment:
What's the use case for multi-puzzle bulk hinting?

Reply:
PH22 is planning to distribute response/hint editing across fewer people than module hunts have traditionally used, so it's a timesaver there..